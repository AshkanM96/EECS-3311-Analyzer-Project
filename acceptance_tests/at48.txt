equals
open_set_enumeration
add
integer_constant (2)
integer_constant (3)
integer_constant (7)
close_set_enumeration
open_set_enumeration
integer_constant (5)
integer_constant (7)
close_set_enumeration
evaluate
type_check
open_set_enumeration
