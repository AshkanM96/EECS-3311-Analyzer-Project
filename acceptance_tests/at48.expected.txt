  Expression currently specified: ?
  Report: Expression is initialized.
->equals
  Expression currently specified: (? = nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({?} = nil)
  Report: OK.
->add
  Expression currently specified: ({(? + nil)} = nil)
  Report: OK.
->integer_constant(2)
  Expression currently specified: ({(2 + ?)} = nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({(2 + 3), ?} = nil)
  Report: OK.
->integer_constant(7)
  Expression currently specified: ({(2 + 3), 7, ?} = nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({(2 + 3), 7} = ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({(2 + 3), 7} = {?})
  Report: OK.
->integer_constant(5)
  Expression currently specified: ({(2 + 3), 7} = {5, ?})
  Report: OK.
->integer_constant(7)
  Expression currently specified: ({(2 + 3), 7} = {5, 7, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({(2 + 3), 7} = {5, 7})
  Report: OK.
->evaluate
  Expression currently specified: ({(2 + 3), 7} = {5, 7})
  Report: True
->type_check
  Expression currently specified: ({(2 + 3), 7} = {5, 7})
  Report: ({(2 + 3), 7} = {5, 7}) is type-correct.
->open_set_enumeration
  Expression currently specified: ({(2 + 3), 7} = {5, 7})
  Report: Error (Expression is already fully specified).
